packages.html

```diff
@@ -16,7 +16,36 @@
             <div id="profileDiv" class="profile-div"></div>
         </div>
         <div id="mainContentContainer" class="main-content-container">
-            <div id="navbar" class="navbar-element"></div>
+            <div id="navbar" class="navbar-element">
+              <nav class="navbar navbar-expand-lg navbar-light bg-light">
+                <div class="container-fluid">
+                  <a class="navbar-brand" href="#">Navbar</a>
+                  <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+                    <span class="navbar-toggler-icon"></span>
+                  </button>
+                  <div class="collapse navbar-collapse" id="navbarSupportedContent">
+                    <ul class="navbar-nav me-auto mb-2 mb-lg-0">
+                      <li class="nav-item">
+                        <a class="nav-link active" aria-current="page" href="index.html">Overview</a>
+                      </li>
+                      <li class="nav-item">
+                        <a class="nav-link" href="repositories.html">Repositories</a>
+                      </li>
+                      <li class="nav-item">
+                        <a class="nav-link" href="projects.html">Projects</a>
+                      </li>
+                      <li class="nav-item">
+                        <a class="nav-link" href="#">Packages</a>
+                      </li>
+                    </ul>
+                    <form class="d-flex">
+                      <input class="form-control me-2" type="search" placeholder="Search" aria-label="Search">
+                      <button class="btn btn-outline-success" type="submit">Search</button>
```

Comment:
Let's change this type to `button` to start out because currently if you click it, it refreshes the page.